libs/eo/origin-of-energy/shell/src/lib/eo-origin-of-energy-shell.component.ts

```diff
@@ -14,45 +14,160 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { CommonModule } from '@angular/common';
 import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
-import { EoInlineMessageScam } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
-import { WattIconModule, WattIconSize } from '@energinet-datahub/watt';
-
+import { MatCardModule } from '@angular/material/card';
+import { EoMediaModule } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
+import { EoPieChartScam } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
-  selector: 'eo-metering-points-shell',
+  selector: 'eo-origin-of-energy-shell',
   styles: [
     `
       :host {
         display: block;
+        max-width: 1040px; /* Magic UX number */
+      }
+
+      .chart-row {
+        display: flex;
+        margin-bottom: var(--watt-space-l);
+      }
+
+      .chart-card {
+        max-width: 584px; /* Magic UX number */
+        width: 100%;
+      }
+
+      .chart-box {
+        margin-top: var(--watt-space-m);
+        margin-right: calc(var(--watt-space-xl) - var(--watt-space-s));
+        margin-left: calc(var(--watt-space-xl) - var(--watt-space-s));
+      }
+
+      .chart-tips {
+        width: 360px; /* Magic UX number */
+        gap: var(--watt-space-l);
+        display: flex;
+        flex-direction: column;
+      }
+
+      .global-goals-box {
+        border: 1px solid #f9d557; /* Color not yet added to Watt */
+        max-height: 100px; /* Magic UX number */
+
```

Comment:
I agree :) But the issue here is that the text that UX wants in here, is actually 104px tall. If i take that and add the 1px on top and buttom, the height would be 106 in total.

I will have a talk with UX about it, but otherwise i need to "squish" the box size for this paragraph in order for it to fit :)

Reply:
UX has said i should allow it to grow, so i will fix it to be 106px tall :)